webpack/ForemanTasks/Components/TasksDashboard/Components/StoppedTasksCard/StoppedTasksCardHelper.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { capitalize } from 'lodash';
+import classNames from 'classnames';
+import { Icon } from 'patternfly-react';
+import {
+  TASKS_DASHBOARD_AVAILABLE_QUERY_STATES,
+  TASKS_DASHBOARD_AVAILABLE_QUERY_MODES,
+} from '../../TasksDashboardConstants';
+
+const resultIcons = {
+  error: <Icon type="pf" name="error-circle-o" />,
+  warning: <Icon type="pf" name="warning-triangle-o" />,
+  success: <Icon type="pf" name="ok" />,
+};
+
+export const getStoppedTable = (data, query, time, updateQuery) =>
```

Comment:
I agree and love the fact it is separated now from the Card component.
Since it returns markup, imo it should be a component called `StoppedTable`.

Reply:
You're right, changed